src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -465,6 +468,10 @@ auto_flash(xrt_core::device_collection& deviceCollection)
     std::cout << "****************************************************\n";
     std::cout << "Cold reboot machine to load the new image on device(s).\n";
     std::cout << "****************************************************\n";
+  } else if (need_warm_reboot) {
+    std::cout << "****************************************************\n";
+    std::cout << "Warm reboot machine to load the new SC image on device(s).\n";
```

Comment:
Wording should be improved.  Something like:
"Warm reboot of the device is required to recognize new SC image."